dot/utils_test.go

```diff
@@ -398,6 +398,9 @@ func TestWriteConfig(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			got := writeConfig(tt.args.data, tt.args.fp)
```

Comment:
nit isn't `writeConfig` then just `os.WriteFile` really?? Do we need such `writeConfig` function?

Reply:
Good point. I've removed `writeConfig` function.